tests/scripts/kubeadm.sh

```diff
@@ -64,8 +111,43 @@ case "${1:-}" in
   clean)
     kubeadm_reset
     ;;
+  install)
+    if [ "$#" -lt 2 ]; then
+        echo "invalid install args"
+        usage
+        exit 1
+    fi
+    sudo sh -c "${scriptdir}/kubeadm-install.sh ${KUBE_VERSION}" root
+    case "${2:-}" in
+        master)
+            install_master
+        ;;
+        node)
+            if [ "$#" -eq 5 ]; then
+                install_node $3 $4 $5
+            else
+                echo "invalid number of arguments for install node"
+                usage
+                exit 1
+            fi
+        ;;
+        *)
+            echo "invalid number of arguments for install" >&2
```

Comment:
Perhaps just "invalid argument for install" or something like that, as the number of arguments isn't the issue here.

Reply:
fixed